src/controllers/auth.controller.js

```diff
@@ -54,6 +141,53 @@ class AuthCtrl extends BaseCtrl {
       }
     )(req, res, next)
   }
-}
+  @post('/google_login')
+  async google_login(req, res) {
```

Comment:
function name is camel so change to `googleLogin`

Reply:
sure